### DISCUSSION TITLE
How to remove a specific version

### DISCUSSION THREAD

**Discussion by mimikun** [MordechaiHadad/bob#212] [Q&A] [answered]
My `bob list` result it:

```fish
❯ bob list
┌───────────┬─────────────┐
│  Version  │  Status     │
├───────────┼─────────────┤
│  48c2ad5  │  Installed  │
│  0f4f7d3  │  Installed  │
│  nightly  │  Installed  │
│  v0.10.0  │  Used       │
│  0e187fe  │  Installed  │
└───────────┴─────────────┘
```

I want to delete everything except `nightly`, `head`, `latest` and `v0.10.0`.
What should I do?
For example, is there a way to extract and output only the version like `docker images -q`?

**Comment by MordechaiHadad** [comment, marked as answer]
Alo, head isn't exactly supported "per say" since neovim always has new commits. but simply run `bob rm` and click space on all versions you want to uninstall and click enter
